tcms/testruns/templates/testruns/get.html

```diff
@@ -311,16 +311,28 @@ <h5><span class="test-executions-count"></span> Results</h5>
 
                                             </div>
                                         </div>
-                                        <div class="col-md-3">
-                                            <span>{% trans 'Test Case Attachments' %}:</span>
-                                            <ul class="list-group list-unstyled test-case-attachments">
-                                                <template id="attachments-list-item">
-                                                    <li class="list-group-item'">
-                                                        <a></a>
-                                                    </li>
-                                                </template>
-                                                <li class="list-group-item hidden">{% trans "No attachments" %}</li>
-                                            </ul>
+                                        <div class="col-md-6">
+                                            <div>
+                                                <h4> {% trans "Comments "%} </h4>
+                                                <input class="simplemde-file-upload" type="file" style="display: none">
+                                                <textarea type="text" class="comment-text" name="comment" class="form-control"></textarea>
+                                                <div class="form-group" style="padding: 15px; text-align: right;">
+                                                    <button class="btn btn-default cancel-comment">{% trans "Cancel" %}</button>
+                                                    <button class="btn btn-primary post-comment">{% trans "Post" %}</button>
```

Comment:
leave only 1 button with name Save to be consistent with the TP page. Instead of the Cancel button there will be buttons for statuses later

Reply:
Use "Save" for button text

Reply:
Drop btn-primary class